contracts/chain-adapters/Arbitrum_CustomGasToken_Adapter.sol

```diff
@@ -240,7 +250,17 @@ contract Arbitrum_CustomGasToken_Adapter is AdapterInterface, CircleCCTPAdapter
                 return divFloor;
             }
         } else {
-            return amount * 10**(nativeTokenDecimals - 18);
+            return amount * 10**(NATIVE_TOKEN_DECIMALS - 18);
+        }
+    }
+
+    function _fromNativeTo18Decimals(uint256 amount) internal view returns (uint256) {
```

Comment:
This was taken from [here](https://github.com/OffchainLabs/nitro-contracts/blob/fbbcef09c95f69decabaced3da683f987902f3e2/src/bridge/ERC20Inbox.sol#L152). I removed the check for uint256 overflow, since this shouldn't happen assuming the constructors are correct, which we can easily verify before enabling anything in the hub pool. Also, the scaling is taken from [here](https://github.com/OffchainLabs/nitro-contracts/blob/fbbcef09c95f69decabaced3da683f987902f3e2/src/libraries/DecimalsConverterHelper.sol#L20).

Reply:
So just to note, in the case where `NATIVE_TOKEN_DECIMALS > 18`, we'll be dividing and effectively rounding down the result. Is that OK when passing in this rounded down value into the Inbox.createRetryableTicket?